reader_test.go

```diff
@@ -2159,18 +2148,8 @@ func testACHFileTooLongErr(t testing.TB) {
 	r := NewReader(f)
 	_, err = r.Read()
 
-	if el, ok := err.(base.ErrorList); ok {
-		if p, ok := el.Err().(*base.ParseError); ok {
-			if e, ok := p.Err.(*FileError); ok {
-				if e.Msg != msgFileTooLong {
-					t.Errorf("%T: %s", e, e)
-				}
-			}
-		} else {
-			t.Errorf("%T: %s", el, el)
-		}
-	} else {
-		t.Errorf("No error even though file was too long")
+	if !Has(err, FileTooLongErr("file too long")) {
```

Comment:
This defiantly cleans up the test cases!

Reply:
I like how this cleans up the tests!

Reply:
I'm ok with the changes @Adam?

For the error on testACHFileRead3,  It's a malformed file (intentionally).  It looks like I had it checking three different errors in the file one from read() and two from File.Validate(), so that cn be done or at this point removing the test is fine as it shouldn't decrease reader.go test coverage.

I haven't looked into TestFileEntryDetail so if you have time, you can but I'm guessing it probably needs checks for the 3 errors EntryDetail Record Error, FileHeader Record Error and FileControl Record Error.